administrative-sdk/connection/connection-controller.js

```diff
@@ -22,48 +22,19 @@ module.exports = class Connection {
     this._analysisId = null;
     this._recordingId = null;
     this._recognitionId = null;
-
-    // The addEventListener interface exists on object.Element DOM elements.
-    // However, this is just a simple class without any relation to the DOM.
-    // Therefore we have to implement a pub/sub mechanism ourselves.
-    // See:
-    // https://developer.mozilla.org/en-US/docs/Web/API/EventTarget.addEventListener
-    // http://stackoverflow.com/questions/10978311/implementing-events-in-my-own-object
-    this.events = {};
+    this.emitter = ee({});
 
     const self = this;
     this.addEventListener = function(name, handler) {
```

Comment:
Just partially related, but this construct can be written as a simple method instead of this hackish assignment in the constructor.